### DISCUSSION TITLE
blog/our-new-blog/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#177] [Page comments]
# blog/our-new-blog/

If you read this, chances are our blog is now up! We are proud to annouce that we got a new blog for Scratch Addons.

https://scratchaddons.com/blog/our-new-blog/

**Comment by Hans5958** [comment]
First comment on this site, ever! v1.25.0 should have been released at the time of writing this.

EDIT: Oh yeah, if you are a SA enthusiast and wanted to write on the blog, tell us. We may consider.

**Reply by PetitScratchCat** [reply]
ok

**Comment by JackyBorderCollie** [comment]
It's awesome that Scratch Addons have a blog! Also, second comment on this site.

**Reply by CST1229** [reply]
Actually third because I commented on the docs.

**Comment by Secret-chest** [comment]
third

**Comment by imdevarsh** [comment]
Epic! What kind of things will go here??

**Reply by Hans5958** [reply]
It could be anything, really, but most times Scratch Addons, I presume.

**Comment by CST1229** [comment]
> but expect updates, tips, and other things **in the next days**

Blog: *absolutely nothing for months*

**Comment by iqnite** [comment]
Blog is quite inactive, isn't it?

**Comment by rimopa** [comment]
What the does the "1 minute" mean?
![image](https://user-images.githubusercontent.com/97112832/179853885-1361b886-e121-4874-adff-1f274d4578cf.png)

**Reply by WorldLanguages** [reply]
The time it takes to read

**Reply by Secret-chest** [reply]
> The time it takes to read

That's a stupid thing. The time differs from person to person. Why not just say the word or paragraph count?

**Reply by Hans5958** [reply]
It is at best an estimation, and estimates don't have to be good. Reading time is the best estimates for this kind of text.

A word count is a number that can't be easily comprehend compared to the reading time. 3 short paragraph and 3 long paragraph would take different time, but still 3 paragraph.

**Reply by Wjplatformer** [reply]
I took 5 seconds

**Comment by BroJac5246** [comment]
Is this even going to be used? So far this is the _only_ post. v1.28.0 is about to release- that could be something good to post about.

**Reply by rimopa** [reply]
1.28 is going to be a huge update, but I don't really see how we could make a blog post about it.
  If we do it, we should also do it with every next big update.

**Reply by BroJac5246** [reply]
That's probably what we should do- what else would we write about?

**Reply by rimopa** [reply]
I agree. Now we need someone to write it.

**Reply by BroJac5246** [reply]
I would, but I'm not the best at writing...

I might be willing to help, though.

**Comment by CatLIEda1st** [comment]
I wonder if there is a way they can make a way to skew things easier in the costume editor.

**Comment by CST1229** [comment]
![](https://user-images.githubusercontent.com/68464103/204811362-463d21b4-3e09-4d19-8372-172c86de09b8.png)

**Reply by iqnite** [reply]
I am 50% robot, 50% pizza, 0% human!

**Comment by iqnite** [comment]
Hey I am 50% robot, 50% pizza, 0% human!

**Comment by Ronaldomario** [comment]
Oi gente. Eu sou Ronaldo mario um animador no YouTube que tem um sonho de cria uma serie famosa na internet!

**Comment by Sonic88688** [comment]
Hello. I am Sonic88688. For some reason, whenever I got on Scratch & reset it (because it crashed) & then Scratch Addons basically crashed. I reinstalled Scratch Addons, & STILL it couldn't work! So, I gave up. So, if there is something that can help me, please let me know. Thank you.

**Reply by WorldLanguages** [reply]
Please create another Chrome profile, try installing and using Scratch Addons there and send a screen recording showing the crash.

**Reply by Sonic88688** [reply]
Well, how can I send a recording showing it?

**Reply by rimopa** [reply]
Record your screen using some external program like OBS and attach the file to a comment here

**Reply by WorldLanguages** [reply]
Nowadays, there are websites that allow you to record your screen.

**Reply by Sonic88688** [reply]
Ok. Give me an example of one.

**Reply by rimopa** [reply]
https://www.screencapture.com/

**Comment by rimopa** [comment]
We could do something like Addon of the Week and explain addons in depth.

**Reply by BroJac5246** [reply]
Great idea!

**Reply by WorldLanguages** [reply]
We plan to do this for all addons in an "addon wiki".
Still, we could feature one wiki article weekly/biweekly/monthly.

**Reply by michaeleldar** [reply]
I know this is a bit old, but I wouldn't mind doing the weekly addon thing.

**Comment by CST1229** [comment]
Maybe showcase upcoming new-and-exciting addons (like the improved block injection which probably won't be in 1.30).

**Comment by eyaini** [comment]
I just added this extension to my browser and I love it!
